Source/UZKFileInfo.m

```diff
@@ -65,6 +97,24 @@ - (NSDate *)timestamp {
 
 
 
+#pragma mark - Private Class Methods
+
+
++ (BOOL)itemIsDirectory:(unz_file_info64 *)fileInfo {
+    UZKZipOS zipOSClassMapping = fileInfo->version >> 8;
```

Comment:
Let's rename this variable `zipOS`, and maybe also encapsulate each dereference and bit manipulation operation for `fileInfo` into a separate private class method. In this case, add:

    + (UZKZipOS) itemOS:(unz_file_info64 *)fileInfo { ... }